doc/source/performance.rst

```diff
@@ -0,0 +1,100 @@
+.. _performance_considerations:
+
+.. currentmodule:: pandas
+
+.. ipython:: python
+   :suppress:
+
+   import pandas as pd
+   import numpy as np
+   np.random.seed(123456)
+
+******************
+Performance Considerations
+******************
+
+Overview
+--------
+Pandas, for most use cases, will be fast; however, there are a few
+anti-patterns to avoid. The following is a guide on achieving better
+performance with pandas.
+
+.. ipython:: python
+
+   import sys
+
+   import numpy as np
+   import pandas as pd
+   import matplotlib.pyplot as plt
+
+   pd.options.display.max_rows = 10
```

Comment:
Changing this may mess with the formatting else where (I think it's all a global state). Maybe just remove it for now and see how things look.